src/init/raw_data.rs

```diff
@@ -9,31 +9,28 @@ use crate::utilities::{basename, download, filename, print_count, untar, unzip};
 use crate::{County, Input, MSOA};
 
 pub struct RawDataResults {
-    pub tus_files: Vec<String>,
+    pub population_files: Vec<String>,
```

Comment:
Also renaming these to match the new input files Hadrien made. `tus` was time use survey, for the record